content/docs/pipelines/pipelines-deploy-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Deploy Alternatives"
+description = "Alternative approaches to deploy lightweight Kubeflow Pipelines to a cluster"
+weight = 15
++++
+
+Despite [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+Kubeflow Pipelines' lite deployment. Pipelines has a kustomize manifests for
+this. You can follow the instructions below to deploy it in an existing cluster.
+
+Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
+document better and be able to customize based on your needs.
+
+
+## TL;DR
+
+To deploy Kubeflow Pipelines in an existing Kubernetes cluster, do the following:
+
+1. Deploy latest version of Kubeflow Pipelines
+```
+export PIPELINE_VERSION={{% kfp-latest-version %}}
+kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/namespaced-install.yaml
+```
+
+2. Get the Pipeline UI URL
+```
+kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com 
+```
+
+## Step by step tutorial to deploy Pipelines from scratch
+
+1. Prepare a Kubernetes cluster
+
+    You have many options from cloud service providers. For GCP, you can reference
+    [Creating a cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster).
+
+    You can also set up your own Kubernetes cluster.
+
+    Cluster minimum requirement: node pool with at least 2 nodes and 2 CPUs per node.
+
+1. Download kubectl CLI tool
+
+    You can get kubectl from [its official doc](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+1. Configure kubectl to talk to your cluster
+
+    For GCP, you can reference [Configuring cluster access for kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl)
+
+1. Deploy latest version of Kubeflow Pipelines Lite to your cluster
+
+    ```
+    export PIPELINE_VERSION={{% kfp-latest-version %}}
+    kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+    ```
+
+    Kubeflow pipelines applications take a while (~3 minutes) to start.
+
+1. Get public URL of Pipelines UI and use it to use Kubeflow Pipelines
```

Comment:
Suggested wording:
"Get the public URL of the Kubeflow Pipelines UI and use it to access Kubeflow Pipelines:"